spec/chatbox.js

```diff
@@ -1240,6 +1240,33 @@
                     });
                 }));
 
+                it("will render the message time as configured", mock.initConverse(function (_converse) {
+                    test_utils.createContacts(_converse, 'current');
+                    test_utils.openControlBox();
+                    test_utils.openContactsPanel(_converse);
+
+                    _converse.time_format = 'hh:mm';
+                    spyOn(_converse, 'emit');
+                    var contact_jid = mock.cur_names[0].replace(/ /g,'.').toLowerCase() + '@localhost';
+                    test_utils.openChatBoxFor(_converse, contact_jid);
+                    expect(_converse.emit).toHaveBeenCalledWith('chatBoxFocused', jasmine.any(Object));
+                    var view = _converse.chatboxviews.get(contact_jid);
+                    var message = 'This message is sent from this chatbox';
+                    spyOn(view, 'sendMessage').andCallThrough();
+                    test_utils.sendMessage(view, message);
+                    expect(view.sendMessage).toHaveBeenCalled();
+                    expect(view.model.messages.length, 2);
```

Comment:
The two `expect` lines above are irrelevant for this test and should be removed. Also the `spyOn` above that.